deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/Utils.java

```diff
@@ -15,4 +15,13 @@ static String decapitalize(String str) {
       return str;
     }
   }
+
+  static boolean hasEmptyOrNullString(String[] strings) {
+    for (String s : strings) {
+      if (s == null || s.isEmpty()) {
```

Comment:
`TextUtils` is part of the android sdk. This is a pure java library, so we don't have access to it